migration/migrator/migrations/master/20230914092120_add_to_name_to_email_database.py

```diff
@@ -0,0 +1,34 @@
+"""Migration for the Submitty master database."""
+
+
+def up(config, database):
+    """
+    Run up migration.
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    """
+    database.execute("""
+        ALTER TABLE emails
+        ADD COLUMN to_name VARCHAR;
+        --- Now user_id null if to_name used instead.
+        ALTER TABLE emails
+        ALTER COLUMN user_id DROP NOT NULL;
+        ALTER TABLE emails
+        ADD CONSTRAINT name_or_email CHECK (
+            (user_id is NOT NULL)
+            <>
+            (to_name is NOT NULL)
+        );
```

Comment:
This migration is not runnable twice. You will need to add some things like `IF NOT EXISTS` or add some stuff to your down migration.

Reply:
Fixed!